### ISSUE TITLE
PyTroch version used?

### ISSUE THREAD

**Issue by Cpruce** [SforAiDl/Neural-Voice-Cloning-With-Few-Samples#15]
I've tried the latest torch (1.1.0) and torch==0.4.0, but I hit NotImplementedError 

```
dataloader_prepared
Model Built!
Los event path: log/run-test2019-05-13_00:22:01.709047
0it [00:00, ?it/s]
Traceback (most recent call last):
  File "speaker_adaptation.py", line 1033, in <module>
    train_seq2seq=train_seq2seq, train_postnet=train_postnet)
  File "speaker_adaptation.py", line 724, in train
    input_lengths=input_lengths)
  File "/home/cory/anaconda3/envs/neural-vc/lib/python3.6/site-packages/torch/nn/modules/module.py", line 491, in __call__
    result = self.forward(*input, **kwargs)
  File "/media/cory/ssd_dsk/Neural-Voice-Cloning-With-Few-Samples/dv3/deepvoice3_pytorch/__init__.py", line 79, in forward
    text_positions, frame_positions, input_lengths)
  File "/home/cory/anaconda3/envs/neural-vc/lib/python3.6/site-packages/torch/nn/modules/module.py", line 491, in __call__
    result = self.forward(*input, **kwargs)
  File "/media/cory/ssd_dsk/Neural-Voice-Cloning-With-Few-Samples/dv3/deepvoice3_pytorch/__init__.py", line 124, in forward
    speaker_embed=speaker_embed, lengths=input_lengths)
  File "/home/cory/anaconda3/envs/neural-vc/lib/python3.6/site-packages/torch/nn/modules/module.py", line 491, in __call__
    result = self.forward(*input, **kwargs)
  File "/media/cory/ssd_dsk/Neural-Voice-Cloning-With-Few-Samples/dv3/deepvoice3_pytorch/deepvoice3.py", line 310, in forward
    text_pos_embed = self.embed_keys_positions(text_positions, w)
  File "/home/cory/anaconda3/envs/neural-vc/lib/python3.6/site-packages/torch/nn/modules/module.py", line 491, in __call__
    result = self.forward(*input, **kwargs)
  File "/media/cory/ssd_dsk/Neural-Voice-Cloning-With-Few-Samples/dv3/deepvoice3_pytorch/modules.py", line 51, in forward
    return self._backend.Embedding.apply(
  File "/home/cory/anaconda3/envs/neural-vc/lib/python3.6/site-packages/torch/nn/backends/backend.py", line 10, in __getattr__
    raise NotImplementedError
NotImplementedError
```

I've been able to run deepvoice3_pytorch from the original repo and used that environment with no luck. Issue seems to stem from `self._backend.Embedding`

What is the recommended pytorch version?

**Comment by phypan11** [comment]
same here, any suggestions?

**Comment by yyg192** [comment]
I tried several versions and finally managed to run the model.
Pytorch=0.3.1
Tensorflow=1.4.1
Cuda=9
Cudnn=7
Python=3.6
My English is poor, I hope it will not cause ambiguity.

I tried a version of pytorch 0.4 or higher, which can be run in deepvoice3_pytorch project, but it cannot be run in the Neural voice project. Then the pytorch version 0.3 can be run on the Neural voice project, but it can't be run in the deepvoice3_pytorch project, which makes me feel confused.
